osf_tests/factories.py

```diff
@@ -534,7 +533,7 @@ class Meta:
 
     @classmethod
     def _create(cls, *args, **kwargs):
-        from osf_tests.utils import get_default_test_schema
+        from osf_tests.utils import get_default_metaschema
```

Comment:
this is just a circular import that's grabbing the locally-defined function. From osf_tests.utils.py:
```
from .factories import (
    get_default_metaschema,
    RegistrationProviderFactory,
    DraftRegistrationFactory,
)
```